### PR TITLE
Add Finnish translation

### PR DESCRIPTION
**Pull request by Lam4hound**
Thank you for this beautiful skin!

Here is the Finnish translation.

### CODE REVIEW

File: language/resource.language.fi_fi/strings.po [hunk 1]
```diff
@@ -0,0 +1,2420 @@
...
+msgstr "Tekstikappaleen automaattinen vieritys"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31247"
+msgid "Colour highlight"
+msgstr "Värillinen korostus"
+
+msgctxt "#31248"
+msgid "Building includes"
+msgstr "Rakennetaan valikoita"
```
Comment (jurialmunkey) [inline language/resource.language.fi_fi/strings.po]: "includes" should be left untranslated as it refers to the `Includes.xml` skin files

File: language/resource.language.fi_fi/strings.po [hunk 2]
```diff
@@ -0,0 +1,2420 @@
...
+
+#: /1080i/Includes_Actions.xml
+msgctxt "#31558"
+msgid "Signpost"
+msgstr "Juliste ja teksti"
+
+#: /1080i/Includes_Views_List.xml
+msgctxt "#31559"
+msgid "List Details"
+msgstr "Lista Tiedot"
```
Comment (jurialmunkey) [inline language/resource.language.fi_fi/strings.po]: I don't speak Finnish but should this be `Lista (tiedot)` instead? 

To match the way you've presented the other viewtype names like `Seinä (tiedot)` and `Rivi (tiedot)` etc.

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
Thanks! This looks good. Just a couple of very minor changes.

**Comment by Lam4hound** [comment]
Wow, that was fast. Thank you.

**Comment by jurialmunkey** [comment]
> Wow, that was fast. Thank you.

I was commenting on another issue and saw the notification pop-up. Thanks for the translation!

**Pull request merged**